### PR TITLE
STCOM-783 After the browser window is minimized the pane is not adjusted so the content on the righthand side is cut

### PR DESCRIPTION
**Pull request by uladislausamets**
## Purpose

https://issues.folio.org/browse/STCOM-783

### CODE REVIEW

File: lib/Paneset/PaneResizeContainer.js [hunk 1]
```diff
@@ -104,21 +98,21 @@ const PaneResizeContainer = ({ isRoot, children, onElementResize, parentElement
         setActiveHandle(null);
       }
     }
-  }, [dragging, activeHandle, onElementResize, cursorX, isRoot]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dragging, activeHandle, onElementResize, cursorX, isRoot, containerRect.current]);
```
Comment (aliaksei-chumakou) [inline lib/Paneset/PaneResizeContainer.js:102]: could you please check this issue, it looks like depending on refs might cause troubles https://github.com/facebook/react/issues/14387

**Comment by ViktorSoroka07** [comment]
I checked it from the branch and works perfectly. Thanks @uladislausamets. I do share concern from @aliaksei-chumakou  though. Aside from that, looks great.

File: lib/Paneset/PaneResizeContainer.js (hunk 1 above)
Comment (uladislausamets) [reply lib/Paneset/PaneResizeContainer.js]: @aliaksei-chumakou @ViktorSoroka07 I removed ref dependency and it works the same now

**Review by aliaksei-chumakou — APPROVED** [review]

**Review by ViktorSoroka07 — APPROVED** [review]

**Review by JohnC-80 — APPROVED** [review]
I like the approach of using hooks/memoization vs debouncing. Looks good!

**Pull request merged**